index.html

```diff
@@ -9,6 +9,26 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="book">
+    <div class="row">
+        <h1 class="title">Título del libro</h1>
```

Comment:
You can have more than one title in your page. We can this class more specific by adding the components name as prefix, like `book-title`